src/Components/SelectionBox.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useContext } from "react";
+import { COLORS } from "../constants";
+import GameContext from "../Contexts/gameContext";
+import { selectColor } from "../Reducers/actions";
+import ColorCirle from "./ColorCircle";
+
+const SelectionBox = () => {
+  const { state, dispatch } = useContext(GameContext);
+
+  return (
+    <div className="selection-box">
+      {COLORS.map((color, colorIndex) => {
+        return (
+          <ColorCirle
+            key={color}
+            style={
+              colorIndex === state.selectedColor
+                ? { backgroundColor: color, border: "2px solid black" }
+                : { backgroundColor: color }
+            }
+            handleClick={() => {
+              dispatch(selectColor(colorIndex));
+            }}
+            isActive={true}
```

Comment:
what is the use of this prop?

Reply:
The colorCircle component is used for both selectionBox and guesses section so isActive prop is true it will handle the click or click would be disabled.
Here it is true because it is of the part of selectionBox which always have to handle the click.
The false part is only for the disabled guesses which are not active.